cmake/Modules/FindComputeCpp.cmake

```diff
@@ -232,13 +238,18 @@ endfunction()
 #  target and sets a dependancy on that new command.
 #
 #  targetName : Name of the target to add a SYCL to.
-#  sourceFile : Source file to be compiled for SYCL.
 #  binaryDir : Intermediate directory to output the integration header.
+#  sourceFiles : Source files to be compiled for SYCL.
 #
-function(add_sycl_to_target targetName sourceFile binaryDir)
+function(add_sycl_to_target targetName binaryDir sourceFiles)
```

Comment:
I forgot to discuss this change. I don't think that one can pass an undefined number of source files without placing them at the very end, unless we explicitly group arguments in the call, i.e.:
https://cmake.org/cmake/help/v3.0/module/CMakeParseArguments.html
Then the order of arguments does not matter but this solution would break the compatibility as well.

If you see another way of implementing it, please share it with me. Otherwise, I should fix my branch by changing CMakeLists for examples.